Aikido.Zen.DotNetCore/Middleware/ContextMiddleware.cs

```diff
@@ -113,9 +113,15 @@ internal string GetParametrizedRoute(HttpContext context)
             // Use the .NET core route collection to match against the request path,
             // ensuring the routes found by Zen match those found by the .NET core
             var routePattern = context.Request.Path.Value;
+
             if (context.Request.Path == null)
             {
-                return string.Empty;
+                return "/";
+            }
+
+            if (RouteHelper.PathIsSingleSegment(context.Request.Path.Value))
```

Comment:
same here

Reply:
I'm also a bit confused what the input is here, wasnt the idea to check the parameterized route, so to not check the route stored in context but the route pattern stored in the routing table?